Checking if logic is correct for banks

sohnshaik opened the pull request:
please review bank por favor

sohnshaik committed:
main framework for idea is here, some thinking about certain things is required

sohnshaik committed:
made the slowerize more fitting to ideals
banks made comments on how we needed to fix, made fix and waiting for review.

stevehagino commented on src/main/java/frc/robot/Robot.java:
```diff
@@ -365,16 +368,26 @@ private Command driveCommand() {
+                .until(trg_teleopScoreReq)
+                .andThen(drivetrain.applyRequest(getTeleSwerveReq()));
+      }
+      if(slow) {
```
If you reassign driverXVelo/driverYVelo in this if statement, you don't need to duplicate the return statement

sohnshaik committed:
slowerization works; need slew rate limiter
TESTED: everything works as intended, just need to implement a slew rate limiter that way robot wont tip when going to L3/4 fast.
